### PR TITLE
multilingual improvements for parler

### PR DESCRIPTION
**Pull request by andimarafioti**
Improving multilingual capabilites for parler

### CODE REVIEW

File: TTS/parler_handler.py [hunk 1]
```diff
@@ -149,7 +162,8 @@ def warmup(self):
 
     def process(self, llm_sentence):
         if isinstance(llm_sentence, tuple):
-            llm_sentence, _ = llm_sentence
+            llm_sentence, language_code = llm_sentence
+            self.speaker = WHISPER_LANGUAGE_TO_PARLER_SPEAKER[language_code]
```
Comment (ylacombe) [inline TTS/parler_handler.py:166]:
```suggestion
            self.speaker = WHISPER_LANGUAGE_TO_PARLER_SPEAKER.get(language_code, "en")
```

Comment (andimarafioti) [reply TTS/parler_handler.py:166]: i'll do:
```suggestion
            self.speaker = WHISPER_LANGUAGE_TO_PARLER_SPEAKER.get(language_code, "Jason")
```

File: arguments_classes/parler_tts_arguments.py [hunk 2]
```diff
@@ -60,3 +59,9 @@ class ParlerTTSHandlerArguments:
             "help": "When using compilation, the prompt as to be padded to closest power of 2. This parameters sets the maximun power of 2 possible."
         },
     )
+    use_default_speakers_list: bool = field(
+        default=True,
```
Comment (ylacombe) [inline arguments_classes/parler_tts_arguments.py:63]:
```suggestion
        default=False,
```

**Review by ylacombe — APPROVED** [review]
LGTM!

**Pull request merged**